ee-9/feature-pack/src/main/resources/feature_groups/domain-host-excludes.xml

```diff
@@ -5,70 +5,80 @@
 
     <feature spec="domain.host-exclude">
         <param name="host-exclude" value="WildFly10.0"/>
-        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.core-management&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.discovery&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.elytron&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.fault-tolerance-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.jwt-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;,&quot;org.wildfly.extension.microprofile.openapi-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.reactive-messaging-smallrye&quot;,&quot;org.wildfly.extension.microprofile.reactive-streams-operators-smallrye&quot;]"/>
+        <param name="excluded-extensions" value="[&quot;org.wildfly.extension.clustering.web&quot;,&quot;org.wildfly.extension.core-management&quot;,&quot;org.wildfly.extension.datasources-agroal&quot;,&quot;org.wildfly.extension.discovery&quot;,&quot;org.wildfly.extension.ee-security&quot;,&quot;org.wildfly.extension.elytron&quot;,&quot;org.wildfly.extension.health&quot;,&quot;org.wildfly.extension.metrics&quot;,&quot;org.wildfly.extension.microprofile.config-smallrye&quot;,&quot;org.wildfly.extension.microprofile.fault-tolerance-smallrye&quot;,&quot;org.wildfly.extension.microprofile.health-smallrye&quot;,&quot;org.wildfly.extension.microprofile.jwt-smallrye&quot;,&quot;org.wildfly.extension.microprofile.metrics-smallrye&quot;,&quot;org.wildfly.extension.microprofile.openapi-smallrye&quot;,&quot;org.wildfly.extension.microprofile.opentracing-smallrye&quot;,&quot;org.wildfly.extension.microprofile.reactive-messaging-smallrye&quot;,&quot;org.wildfly.extension.microprofile.reactive-streams-operators-smallrye&quot;,&quot;org.wildfly.extension.opentelemetry-extension&quot;]"/>
```

Comment:
That was done to differentiate between this module and the user-exposed -api module. It _is_ verbose and redundant, so that can probably left off.

Reply:
The opentelemetry-api module is jboss-api=private, so its name is not particularly relevant to users. The extension module name is because its part of the user managed config.